Woof/WoofTests/Model/KeyValueStorage/TestData/KeyValueStorage+TestData.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+extension KeyValueStorage {
+    /// Data for key-value storage to use in unit tests.
+    enum TestData {
+        /// The byte buffer of size 1 byte.
+        static let dataOneByte = Data("A".utf8)
+
+        /// The byte buffer of size 2 bytes.
+        static let dataTwoBytes = Data("AA".utf8)
+
+        /// The byte buffer of size 10 bytes.
+        static let dataTenBytes = Data("AAAAAAAAAA".utf8)
```

Comment:
nit: I didn't find the tests where we use it.

Reply:
Removed, thanks!